src/test/esperanza/finalizationstate_withdraw_tests.cpp

```diff
@@ -100,14 +69,7 @@ BOOST_AUTO_TEST_CASE(process_withdraw_too_early) {
       spy.ProcessVote(vote);
     }
 
-    if (i <= end_logout) {
-      BOOST_CHECK_EQUAL(spy.ValidateWithdraw(validatorAddress, depositSize),
-                        +Result::WITHDRAW_BEFORE_END_DYNASTY);
-    } else {
-      BOOST_CHECK_EQUAL(spy.ValidateWithdraw(validatorAddress, depositSize),
-                        +Result::WITHDRAW_TOO_EARLY);
-    }
```

Comment:
What's wrong with these checks?

Reply:
We removed `WITHDRAW_BEFORE_END_DYNASTY` state. Now there is only one error which means we withdraw too early. If we want to know what is exactly the state of finalizer, we ask FinalizationState.

Reply:
Resolving. Feel free to re-open if you have suggestions for improvements